server/controllers/email-controller.js

```diff
@@ -142,7 +142,7 @@ exports.getListOfLectures = async (from, to) => {
   //let querystring = `select distinct * from lecture join user where lecture.lecturer==user.id and lecture.start>='${date.format("YYYY-MM-DD HH:mm:ss")}' and  lecture.start<'${date.add(1, "days").format("YYYY-MM-DD HH:mm:ss")}' `;
   const queryResults = await knex
     .select(
-      { name: "lecture.name" },
+      //{ name: "lecture.name" },
```

Comment:
Usually it's seen as a bad practice to commit files containing commented lines of code.